source/_data/sessions.yaml

```diff
@@ -52,3 +52,62 @@ items:
         <p>"Compilers" are usually-mysterious things that a lot of us actually use and rely on from day to day, eg. the JavaScript V8 engine+compiler (that Node.js and Google Chrome use to run JavaScript), or the TypeScript compiler. Some parts of compilers can appear in other tools too, such as linters and code formatters.</p>
         <p>Digging into how these work can be satisfying in of itself, like sleuthing a good puzzle, but some of the ways we implement them can appear in other forms, such as when validating user input or writing a wrapper library.</p>
         <p>This session will be kicked off by a quick talk that introduces the main parts of a compiler, followed by a "Stream Adventure"-style guided workshop through building a little compiler of your own</p>
+    -
+      name: Alex Hernandez
+      image: "/images/speakers/ix-alex-hernandez.jpg"
+      description: Product designer and Vue.js fanatic/front-end developer focusing on design systems. Last year, he helped launch Leasecake, the eventual of winner of Techstars Global Startup Weekend 2017. In his spare time, he cultivates communities around Vue.js, remote work, and inclusivity.
+      sessionTitle: Demystifying UI Animations with Vue.js
+      synopsis: <p>This talk showcases examples of different types of animation across the web, from simple button color changes to whimsical&nbsp;<a href="https://codepen.io/OyeOvi/pen/RMMJBy" >character-driven login screens</a>. It will clarify the role and significance of animation in the user experience. And show how to build them in Vue.js via a series of CodePens inspired by <a href="https://uxdesign.cc/good-to-great-ui-animation-tips-7850805c12e5">"Good to great UI animation tips"</a> by Pablo Stanley.</p>
+    -
+      name: Tracy Kaur
+      image: "/images/speakers/ix-tracy-kaur.jpg"
+      jobTitle: Launch engineer
+      employer: Shopify Plus
+      sessionType: workshop
+      description: >
+        Tracy works remotely as a Launch Engineer at Shopify Plus and would like to encourage you all to think like an entrepreneur.
+        Prior to joining Shopify she was an educator at Coder Academy after having just gone through the bootcamp herself! She is a qualified teacher, NodeGirl and has a passion for learning new things, and teaching others what I know.
```

Comment:
Third person suddenly switches to first: "teaching others what she knows"

Reply:
Thanks @AnnaGerber. Fixed it.